pymongo/message.py

```diff
@@ -1243,59 +1259,19 @@ def __batch_command(
         # Chop off the OP_QUERY header to get a properly batched write command.
         cmd_start = msg.index(b"\x00", 4) + 9
         outgoing = _inflate_bson(memoryview(msg)[cmd_start:], DEFAULT_RAW_BSON_OPTIONS)
-        return outgoing, to_send
-
-    def execute(
-        self, cmd: MutableMapping[str, Any], docs: list[Mapping[str, Any]], client: MongoClient
-    ) -> tuple[Mapping[str, Any], list[Mapping[str, Any]]]:
-        batched_cmd, to_send = self.__batch_command(cmd, docs)
-        result: Mapping[str, Any] = self.conn.command(
-            self.db_name, batched_cmd, codec_options=self.codec, session=self.session, client=client
-        )
-        return result, to_send
-
-    def execute_unack(
-        self, cmd: MutableMapping[str, Any], docs: list[Mapping[str, Any]], client: MongoClient
-    ) -> list[Mapping[str, Any]]:
-        batched_cmd, to_send = self.__batch_command(cmd, docs)
-        self.conn.command(
-            self.db_name,
-            batched_cmd,
-            write_concern=WriteConcern(w=0),
-            session=self.session,
-            client=client,
-        )
-        return to_send
+        return -1, outgoing, to_send
 
     @property
     def max_split_size(self) -> int:
         """Reduce the batch splitting size."""
         return _MAX_SPLIT_SIZE_ENC
 
 
-def _raise_document_too_large(operation: str, doc_size: int, max_size: int) -> NoReturn:
```

Comment:
Can you move the new _raise_document_too_large back here to minimize code churn?